index.html

```diff
@@ -264,350 +274,353 @@ <h2>Intro Section of the Document</h2>
-              <li>Collaboration between the custodian, reviewers, and submitters is highly encouraged, ideally through a Plugfest or another structured testing session where different implementations can be evaluated collectively.</li>
+      <section id="submission-requirements">
+        <h2>Submission Requirements</h2><!-- Requirements on the Submitted Document -->
+        
+        <p>What does the binding have to contain to go into the table.</p>
+        <ul>
+          <li><a href="#req-opmap"></a></li>
+          <li><a href="#req-contmap"></a></li>
+          <li><a href="#req-traninit"></a></li>
+          <li><a href="#req-changelog"></a>
+          <li><a href="#req-docsec"></a>
+          <li><a href="#req-copy"></a>
+          <li><a href="#req-openread"></a>
+          <li><a href="#req-openrev"></a>
+          <li><a href="#req-summ"></a>
+          <li><a href="#req-trancurr"></a>
+          <li><a href="#req-content"></a>
+          <li><a href="#req-docs"></a>
+          <li><a href="#req-confl"></a>
+          <li><a href="#req-redef"></a>
+          <li><a href="#req-deps"></a>
+          </li>
+        </ul>
+        
+        <p class="ednote">[DP] Should we create a table instead of the (sub-)sections?</p>
+        
+        
+        <section id="req-opmap">
+          <h3>WoT operation</h3>
+          <p><span class="rfc2119-assertion" id="submission-requirements-wot-operation">A binding that uses a protocol MUST map at least one WoT operation (<code>op</code> keyword value such as <code>readproperty</code>) to a protocol message and vice versa.</span></p>
+        </section>
+        
+        <section id="req-contmap">
+          <h3>ContentType</h3>
+          <p><span class="rfc2119-assertion" id="submission-requirements-contenttype">A binding that uses a serialization format via the <code>contentType</code> keyword MUST mention how the Data Schema terms should be used to describe the messages.</span> This avoids submission of a binding like &quot;XML Binding&quot; that says &quot;Use <code>contentType:application/xml</code> and nothing more. That alone would not be enough to serialize correct messages based on the data schema.</p>
+          <p class="ednote">TODO: We will need additional mechanisms (including vocabulary terms) to ensure that it is possible to use other media types.</p>
+        </section>
+        
+        <section id="req-traninit">
+          <h3>Initial entry</h3>
+          <p><span class="rfc2119-assertion" id="submission-requirements-req-content">Initial entry MUST be a correct document which complies with <a href="#req-content">Req-content</a>.</span> <span class="rfc2119-assertion" id="submission-requirements-map-check">The reviewer MUST NOT check whether the binding tries to map <code>readproperty</code> to a non-existent HTTP method.</span> A successful initial document triggers a &quot;Call for Implementation&quot;.</p>
```

Comment:
The second one with MUST NOT should be reformulated as we did not intend a real assertive language there